diagnostics/diagnostics_metrics.go

```diff
@@ -0,0 +1,95 @@
+package diagnostics
+
+import (
+	"runtime"
+
+	"github.com/stripe/veneur/v14"
+)
+
+var (
+	gitSha                  string = "git_sha: " + veneur.VERSION
+	prevTotalMalloc         uint64 = 0
+	prevTotalFreed          uint64 = 0
+	prevTotalHeapAllocBytes uint64 = 0
```

Comment:
nit: avoid global state.

Reply:
done, keeping global state for just gitsha